db/fixtures/miq_product_features.yml

```diff
@@ -13,7 +13,8 @@
   :children:
   - :name: Accordions
     :description: All Accordions under Workloads
-    :feature_type: node
+    :feature_type: nodecd ../..
```

Comment:
@h-kataria Is this a typo or was it intended to be "nodecd ../.."? It causes the model to blow up. I changed it locally to "node" to get past it.